### ISSUE TITLE
Define encoding for HTTP Basic Authentication

### ISSUE THREAD

**Issue by annevk** [whatwg/fetch#282]
See https://bugzilla.mozilla.org/show_bug.cgi?id=436033 which suggests Chrome always uses utf-8 and Firefox sometimes uses utf-8.

**Comment by mnot** [comment]
FWIW:
  http://tools.ietf.org/html/rfc7617

especially `B.3.  Why not simply switch the default encoding to UTF-8?`

**Comment by annevk** [comment]
Right, except that forgot about market reality, in which the really big user agents make the rules for the rest of the ecosystem.

That is, systems will be coded towards Chrome and Safari and not even consider UTF-8 might not be used. That is much more likely than any kind of user-agent sniffing.

**Comment by mnot** [comment]
If there is convergence (or it emerges), please ping me and we'll get that RFC fixed.
